src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""BKSF transform on fermionic operators."""
+#from __future__ import absolute_import
+
+import numpy as np
+from fermilib.ops import InteractionOperator
+from projectq.ops import QubitOperator
+import networkx as nx
+
+
+def bksf(operator):
+    """ Find the bksf pauli-representation of InteractionOperator. 
+        bksf pauli-representation of general FermionOperator is 
+        not possible in bksf. So the InteractionOperator given as input must
+        be Hermitian.
+    Returns:
+        transformed_operator: An instance of the QubitOperator class.
+
+    """
+    if isinstance(operator, InteractionOperator):
```

Comment:
There are no expression for odd number of fermionic operators (in multiplication) in BKSF. Right now, I have worked out the algebraic expressions for the different terms in the hamiltonian. The general expression (with even number of fermionic operators) might be little complicated, so, let me think about it.